### PR TITLE
Adapt to new Noted's guidelines and new NLP route

### PR DESCRIPTION
**Pull request by edouard-sn**
#### Description

Adapted to [protorepo](https://github.com/noted-eip/protorepo) and other services new guidelines and style
New ExtractKeywordBatch route (as if you were doing multiple request to ExtractKeyword, but only with a list of texts)

#### Changelog

- [ENHANCEMENT] Adapted code and documentation to the new [protorepo](https://github.com/noted-eip/protorepo)'s guidelines (basically changed names, some file location/existence and build steps)
- [ENHANCEMENT] ExtractKeywordBatch with a unit test on a valid request

### CODE REVIEW

File: recommendations_service.py [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import os, sys, json
 
 import grpc
-import recommendationspb.recommendations_pb2_grpc as pb2_grpc
-import recommendationspb.recommendations_pb2 as pb2
+import protorepo.noted.recommendations.v1.recommendations_pb2_grpc as pb2_grpc
```
Comment (maxime-dodin) [inline recommendations_service.py:4]: il nous faut une norme pour les import entre les services
le service Account import le .pb as accountv1

**Review by gabriel654165 — APPROVED** [review]

File: recommendations_service.py (hunk 1 above)
Comment (rojas-diego) [reply recommendations_service.py:4]: Ouai je suis d'accord. C'est pas hyper explicite `pb2_grpc`. Genre `recommendationsv1` ou `recommendations` ou `recommendationspb` ce serait plus explicite je pense.

**Review by rojas-diego — APPROVED** [review]
Trop cool!

File: recommendations_service.py (hunk 1 above)
Comment (edouard-sn) [reply recommendations_service.py]: C'est `recommendationspb(_grpc)` maintenant

**Comment by rojas-diego** [comment]
I think you can merge this!

**Pull request merged**